sunpy/io/tests/test_cdf.py

```diff
@@ -23,3 +23,23 @@ def test_read_cdf():
     assert col.unit == u.Unit("1 / (cm2 MeV s sr)")
     # Check that fillvals are replaced by NaN
     assert np.sum(np.isnan(col)) == 189
+
+def test_check_nan_values():
+    all_ts = read_cdf(filepath)
+    assert isinstance(all_ts, list)
+
+    assert len(all_ts) > 0
+
+    for i, ts in enumerate(all_ts):
+        print(f"Checking Timeseries {i + 1} for NaN values...")
+        has_nan = False
+
+        data = ts.to_dataframe()
+        for column in data.columns:
+            if data[column].isna().any():
+                has_nan = True
+                print(f"  Column '{column}' contains NaN values.")
+
+        if not has_nan:
+            print(f"  No NaN values found in Timeseries {i + 1}.")
+        print()
```

Comment:
This isn't how a test should be written. Please have a read over https://realpython.com/pytest-python-testing/